.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,52 @@
+name: lighthouse
+on:
+  push:
+    branches-ignore:
+      - master
```

Comment:
do we want to ignore master?

How does it know what to compare against

Reply:
I thought we have to add this for other branches so that whenever we merge a branch into master it should be at recommended performance scores. Should I need to change it to all branches?